packages/ui/src/views/repo/repo-summary/repo-summary.types.ts

```diff
@@ -0,0 +1,20 @@
+export enum SummaryItemType {
+  Folder = 0,
+  File = 1
```

Comment:
Totally fine to be specific, but in this case, you can kill the `= 0` and it'll work the same since enums are 0 indexed anyway.
```suggestion
  Folder,
  File
```